README.md

```diff
@@ -1 +1,87 @@
-# Hello-Microverse-Projects
\ No newline at end of file
+# Hello-Microverse-Projects
+logoZ.png
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
```

Comment:
- [ ] Kindly customize this [Microverse README file template](https://github.com/microverseinc/readme-template/blob/master/README.md) nicely. Kindly ensure to remove all the template text and placeholders, they are there to instruct on what to do with each header. **Note: The only optional header is the FAQ.**